presto-docs/src/main/sphinx/connector/localfile.rst

```diff
@@ -0,0 +1,45 @@
+====================
+Local File Connector
+====================
+
+The local file connector allows querying data stored on the local
+filesystem of each worker.
+
+Configuration
+-------------
+
+To configure the local file connector, create a catalog properties file
+``etc/catalog/localfile.properties`` with the following contents:
```

Comment:
This makes it appear as if the file *has* to be named that way. I'd just say something like "create a catalog properties file under etc/catalog (e.g., localfile.properties)"